### ISSUE TITLE
Slow encoding to h264 using software codec from FFMPEG

### ISSUE THREAD

**Issue by romanshvr2n** [stephaniewm55/1-Click-Torrent-Youtube-FFMPEG-Client#2]
Transcoding does work instantly, but encoding to h264 using FFMPEG software codec is slow. Could you add hardware codec?

**Comment by stephaniewm55** [comment]
Yes I plan to add that when I re-implement File Explorer mouse context menu through DLL component instead of Registry entries.
